scripts/init.sh

```diff
@@ -19,7 +19,11 @@ check_for_termination() {
 }
 
 put_metric() {
```

Comment:
I added this to most of our batch jobs in gen-epi. I'm wondering if it's worth making that a ~standard way to point scripts at a different endpoint URL

```
if [ -n "${BOTO_ENDPOINT_URL-}" ]; then
  export aws="aws --endpoint-url ${BOTO_ENDPOINT_URL}"
else
  export aws="aws"
fi

# Example:
# $aws s3 cp s3://foo/bar .
```